tests/testthat/test-parse.R

```diff
@@ -109,7 +127,14 @@ test_that("parse_connect_rfc3339() parses timestamps with offsets as expected",
   expect_identical(parse_connect_rfc3339(single_offset), expected[1])
 })
 
+
 test_that("parse_connect_rfc3339() handles fractional seconds", {
+  original_tz <- Sys.getenv("TZ")
+  withr::defer({
+    Sys.setenv(TZ = original_tz)
+  })
+
+  Sys.setenv(TZ = "UTC")
```

Comment:
Can we use `local_envvar()` for this?

```suggestion
  withr::local_envvar(TZ = "UTC")
```